eth_defi/gmx/base.py

```diff
@@ -49,11 +46,7 @@ class GMXClient:
 
         # Place trades (requires wallet configuration)
         if gmx.has_write_capability():
-            trade_result = gmx.trading.open_position(
-                market="ETH/USD",
-                side="long",
-                size_usd=1000
-            )
+            trade_result = gmx.trading.open_position(market="ETH/USD", side="long", size_usd=1000)
```

Comment:
Can we configure to ruff keep multiline call convention if the developer has opted in for it for the readability?